packages/dashboard/src/components/Wizard/Wizard.types.ts

```diff
@@ -32,6 +36,10 @@ export interface IWizardStepProps {
   wizardContent?: IWizardContentProps;
 
   subSteps?: IWizardStepProps[];
+
+  titleElement: JSX.Element;
+
+  footerElement: JSX.Element;
```

Comment:
are these respected in SetupWizard?

Reply:
Not currently. @ashasunmsft will work on top of this PR and change setup wizard.